xclim/core/units.py

```diff
@@ -896,11 +926,11 @@ def amount2rate(
 
     Parameters
     ----------
-    amount : xr.DataArray, pint.Quantity or string
+    amount : xr.DataArray or pint.Quantity or str
```

Comment:
```suggestion
    amount : xr.DataArray | pint.Quantity | str
```

In other place in xclim we use the pipe, maybe we should uniformize this as well?

Reply:
The spec suggests using plaintext descriptions here: https://numpydoc.readthedocs.io/en/latest/format.html#parameters.